tailcfg/tailcfg.go

```diff
@@ -158,7 +158,8 @@ type CapabilityVersion int
 //   - 111: 2025-01-14: Client supports a peer having Node.HomeDERP (issue #14636)
 //   - 112: 2025-01-14: Client interprets AllowedIPs of nil as meaning same as Addresses
 //   - 113: 2025-01-20: Client communicates to control whether funnel is enabled by sending Hostinfo.IngressEnabled (#14688)
-const CurrentCapabilityVersion CapabilityVersion = 113
+//   - 114: 2025-01-29: Client sends audit-log entries to control
+const CurrentCapabilityVersion CapabilityVersion = 114
```

Comment:
we can't bump this in this PR (which only adds types) because it would misrepresent to control the capabilities of the client. 

But does control actually need to know (in the later PR) when the client is capable of this?